ts/sagas/startup/generateCryptoKeyPair.ts

```diff
@@ -93,6 +97,35 @@ function* generateCryptoKeyPair(keyTag: string) {
   }
 }
 
+/**
+ * Get the public key liked the provided keyTag
+ */
+export function* getCryptoPublicKey(keyTag: O.Option<string>) {
```

Comment:
It's a better practice to handle monadic structure at the caller level and avoid to take them as argument ie. 

`export function* getCryptoPublicKey(keyTag: string) { ... }`

then call it like: 

`maybeKeyTag.fold(() => /* handle empty value *// , getCryptoPublicKey)`

Reply:
As above. See [here](https://pagopa.atlassian.net/browse/IOAPPCIT-48).